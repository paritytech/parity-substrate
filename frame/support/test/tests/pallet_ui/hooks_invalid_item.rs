#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::Hooks;

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T>(_);
}

fn main() {
}
