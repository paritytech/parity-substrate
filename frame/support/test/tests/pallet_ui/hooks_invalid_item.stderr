error[E0107]: missing generics for trait `Hooks`
    --> $DIR/hooks_invalid_item.rs:12:18
     |
12   |     impl<T: Config> Hooks for Pallet<T> {}
     |                     ^^^^^ expected 1 type argument
     |
note: trait defined here, with 1 type parameter: `BlockNumber`
    --> $DIR/traits.rs:2095:11
     |
2095 | pub trait Hooks<BlockNumber> {
     |           ^^^^^ -----------
help: use angle brackets to add missing type argument
     |
12   |     impl<T: Config> Hooks<BlockNumber> for Pallet<T> {}
     |                          ^^^^^^^^^^^^^
